### PR TITLE
[WIP] chore: re-enable API doc generation for angular_test

### LINKED ISSUE #1011: API: re-enable inclusion of angular_test API
Just before releasing the 4.0 docs, we  had dartdoc hang (on Travis, not locally) when attempting to generate the API docs over `angular_test`.

To add `angular_test` back, simply drop the package list given on the command line, here: https://github.com/dart-lang/site-webdev/blob/master/.travis.yml#L23

### PR DESCRIPTION
**Pull request by chalin**
Fixes #1011

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ env:
     - E2E_OPT2="--fast --filter=examples/ng/doc/[t-z]"
     - TEST_OPT=""
   matrix:
-    - CI_TASK="build --dartdoc=acx,forms,ng,router" DART_CHANNEL=stable
+    - CI_TASK="build --dartdoc=ngtest --log-at=info" DART_CHANNEL=stable
```
Comment (kevmoo) [inline .travis.yml]: Don't you just want to add ngtest to the list?

Comment (chalin) [reply .travis.yml]: Sorry, I should have marked this PR WIP from the start (I just did).

It is/was already part of the list. The change above selects _only_ angular_test.

So far, no matter what I try, I am unable to generate the dartdocs for `angular_test` on Travis. I can generate the docs locally w/o any difficulties, but on Travis the job just hangs. I'm using this PR to try to debug the situation.

**Comment by kevmoo** [comment]
Ah! You need this fix -
https://github.com/dart-lang/angular/commit/256a7cca28d2d40a8184def5a566385bf592e207

I think it's due to pub

**Comment by chalin** [comment]
Great that someone put the finger on that! Looking forward to trying the next release. In the meantime, I'll let this rest.

**Comment by chalin** [comment]
Fixed by #1064 (which patches the problematic `pubspec.yaml` entry).

**Pull request closed without merging**